actions/app.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/gopheracademy/gcon/actions/admin"
 	"github.com/gopheracademy/gcon/models"
+	"github.com/kiasaki/gcon/actions/countdown"

```

Comment:
Oops, yes I pull down my fork and `goimports` used the fork's path as the current directory of my editor was `$GOPATH/src/github.com/kiasaki/gcon`